packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -327,11 +330,30 @@ function Preview({ isInspecting, setIsInspecting, zoomLevel, onZoomChanged }: Pr
     device: device!,
   });
 
+  const workspace = useWorkspaceConfig();
+
+  const isFrameDisabled = workspace.showDeviceFrame === false;
+
+  function MaybeFrame() {
+    if (!device) {
+      return null;
+    }
+    return (
+      <img
+        src={device.frameImage}
+        className="phone-frame"
+        style={{
+          opacity: isFrameDisabled ? 0 : 1,
```

Comment:
`opacity` because it still needs to be included in layout calculation?

Reply:
@jakub-gonet yes